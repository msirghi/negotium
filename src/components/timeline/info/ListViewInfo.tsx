import InfoIcon from '@mui/icons-material/Info';
import { Row } from '../../common/utilities/row/Row';
import { makeStyles } from '@mui/styles';
import colors from '../../../common/styles/colors';
import useTranslation from 'next-translate/useTranslation';

const useStyles = makeStyles({
  text: {
    fontSize: 12,
    marginLeft: 3,
    color: colors.greys['500'],
  },
  icon: {
    fontSize: 15,
    color: colors.greys['500'],
  },
});

export const ListViewInfo = () => {
  const classes = useStyles();
  const { t } = useTranslation('timeline');

  return (
    <Row alignVerticalCenter>
      <InfoIcon className={classes.icon} />
      <span className={classes.text}>{t('listView.noteMessage')}</span>
    </Row>
  );
};
