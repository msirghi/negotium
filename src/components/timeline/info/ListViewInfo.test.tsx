import renderer from 'react-test-renderer';
import { ListViewInfo } from './ListViewInfo';

describe('ListViewInfo', () => {
  it('should match the snapshot', () => {
    const component = renderer.create(<ListViewInfo />);
    expect(component).toMatchSnapshot();
  });
});
