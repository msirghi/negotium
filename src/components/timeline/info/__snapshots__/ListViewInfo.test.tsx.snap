// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListViewInfo should match the snapshot 1`] = `
<div
  onClick={[Function]}
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "row",
      "justifyContent": "flex-start",
      "width": "initial",
    }
  }
>
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium makeStyles-icon-2 css-i4bv87-MuiSvgIcon-root"
    data-testid="InfoIcon"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 15h-2v-6h2v6zm0-8h-2V7h2v2z"
    />
  </svg>
  <span
    className="makeStyles-text-1"
  >
    listView.noteMessage
  </span>
</div>
`;
